components/Layout/Page.tsx

```diff
@@ -81,11 +95,16 @@ const GlobalStyles = createGlobalStyle`
   button {
     font-family: --apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
   }
+
+
 `
 
 const InnerStyles = styled.div`
   max-width: 100%;
   margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  flex: 1;
 `
 
 const lightTheme = {
```

Comment:
Why isn't this theme typed with `DefaultTheme`?

ref: https://styled-components.com/docs/api#typescript